src/server/webkit/wkInterceptableRequest.ts

```diff
@@ -126,7 +126,24 @@ export class WKInterceptableRequest implements network.RouteDelegate {
       requestStart: timingPayload ? wkMillisToRoundishMillis(timingPayload.requestStart) : -1,
       responseStart: timingPayload ? wkMillisToRoundishMillis(timingPayload.responseStart) : -1,
     };
-    return new network.Response(this.request, responsePayload.status, responsePayload.statusText, headersObjectToArray(responsePayload.headers), timing, getResponseBody);
+
+    const serverIPAddressAndPort: network.IPAddressAndPort = {
+      ipAddress: '',
+      port: -1,
+    };
+
+    let securityDetails: network.SecurityDetails | undefined;
+    if (responsePayload.security) {
+      securityDetails = {
+        protocol: responsePayload.security.connection?.protocol || '',
+        subjectName: responsePayload.security.certificate?.subject || '',
+        issuer: '',
```

Comment:
WK is the only browser to not have this. It also appears protocol is always blank for WK. I'm happy to drop everything except `validFrom`/`validTo`. Alternatively, I'm happy to pass everything through and `any` type sec details allowing it to be browser specific. (Though that did not feel in the spirit of PW.)